### ISSUE TITLE
Cannot delete wallet

### ISSUE THREAD

**Issue by Sune1337** [goatpig/BitcoinArmory#210]
When i try to delete a wallet the UI silently fails, and the following is logged in the armorylog.txt:

```
2017-04-19 11:14 (ERROR) -- Traceback (most recent call last):
  File ".../qtdialogs.py", line 4518, in <lambda>
    rmWalletSlot = lambda: self.removeWallet(wlt)
  File ".../qtdialogs.py", line 4596, in removeWallet
    self.main.removeWalletFromApplication(wltID)
  File ".../ArmoryQt.py", line 3268, in removeWalletFromApplication
    self.walletMap[wltID].unregisterWallet()
  File ".../armoryengine/PyBtcWallet.py", line 288, in unregisterWallet
    TheBDM.unregisterWallet(self.uniqueIDB58)
  File ".../armoryengine/BDM.py", line 116, in inner
    return func(*newArgs, **kwargs)
  File ".../armoryengine/BDM.py", line 251, in unregisterWallet
    self.bdv().unregisterWallet(uniqueIDB58)
  File ".../CppBlockUtils.py", line 1488, in <lambda>
    __getattr__ = lambda self, name: _swig_getattr(self, BlockDataViewer, name)
  File ".../CppBlockUtils.py", line 57, in _swig_getattr
    raise AttributeError(name)
AttributeError: unregisterWallet
```

It does succeed in deleting the wallet file; if I retry to delete the wallet again i get the following log:
```
2017-04-19 11:23 (INFO) -- qtdialogs.py:4593 - ***Completely deleting wallet
2017-04-19 11:23 (ERROR) -- Traceback (most recent call last):
  File ".../qtdialogs.py", line 4518, in <lambda>
    rmWalletSlot = lambda: self.removeWallet(wlt)
  File ".../qtdialogs.py", line 4594, in removeWallet
    os.remove(thepath)
OSError: [Errno 2] No such file or directory: '.../.armory/armory_xxxxxxx_.wallet'
```

I'm using Version 0.95.1-beta-None on Debian 8.7 based linux.
Built from source: tags/v.0.95.1

**Comment by Sune1337** [comment]
If I restart Armory after trying to delete the wallet, the wallet is gone.

**Comment by goatpig** [comment]
This is fixed in 0.96

**Comment by Sune1337** [comment]
great ;)
